.github/workflows/.windows.yml

```diff
@@ -401,6 +401,11 @@ jobs:
             Start-Sleep -Seconds 1
           }
           Write-Host "Test daemon started and replied!"
+          If ("${{ matrix.runtime }}" -eq "containerd") {
+            If (-not (Get-Process -Name containerd -ErrorAction:SilentlyContinue)) {
+              Throw "containerd process is not running"
+            }
+          }
```

Comment:
If possible, this should ensure we're looking at the right containerd instance. Does `Get-Process` let us pass in a full binary path? My concern is if the underlying runner image (which may have an installed Docker instance as seen in the "Removing current Daemon" step) eventually (as a result of this work or otherwise) may also include a containerd instance.

Another option is to make this test like the test daemon check, and use `ctr version` or something against the expected containerd instance at `\\.\pipe\docker-containerd`, although that would also false-positive against a leaked containerd from system docker... It does have the nice side-effect of putting the containerd version into the CI logs.

Reply:
Get-Process does not directly support filtering processes by their binary paths. However, by using the process ID obtained from Get-Process, you can access and query the binary path.